### PR TITLE
Address screen

### PR DESCRIPTION
**Pull request by johnanisere**
screen shot 2018-06-23 at 9 49 16 pm

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React, { Component } from "react";
 import { Provider } from "react-redux";
-import App from "./Containers/App";
+// import App from "./Containers/App";
 import configureStore from "./configureStore";
 import { createReduxBoundAddListener } from "react-navigation-redux-helpers";
+import App from "./Containers/Address";
```
Comment (jihdeh) [inline src/index.js]: can switch this back

File: .expo/packager-info.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "expoServerPort": 19000,
   "packagerPort": 19001,
-  "packagerPid": 13068,
-  "expoServerNgrokUrl": "https://ft-ud9.jihdeh.juli.exp.direct",
-  "packagerNgrokUrl": "https://packager.ft-ud9.jihdeh.juli.exp.direct",
-  "ngrokPid": 13083
+  "packagerPid": 34044,
```
Comment (jihdeh) [inline .expo/packager-info.json]: seems this file isn't added to gitignore correctly

Comment (johnanisere) [reply .expo/packager-info.json]: it is

Comment (jihdeh) [reply .expo/packager-info.json]: okay i'll handle it

**Comment by jihdeh** [comment]
Resolve the conflicts before this can be merged.
You can delete the yarn.lock file.
Remove .expo from git stage
and check what's wrong with package.json

**Pull request merged**